.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
Actually, I would remove these platform questions for now. Isn't it very unlikely that the parser yields different results on different platforms? If it does, then it's probably an upstream issue and we can still collect this information if necessary :)

Reply:
You're right, of course. That part is copy/paste laziness(;

Having said that, in some projects it might make sense to ask some questions on dev setup. But in org-parser, we're specifying both Clojure and library versions, so only the Java version can differ. 

Like this, it rarely gives rise to trouble. I'm removing the whole section for the time being.